src/main/resources/org/openstreetmap/atlas/checks/database/schema.sql

```diff
@@ -0,0 +1,33 @@
+DROP SCHEMA IF EXISTS {schema} CASCADE;
+CREATE SCHEMA IF NOT EXISTS {schema};
+CREATE EXTENSION IF NOT EXISTS postgis;
+CREATE EXTENSION IF NOT EXISTS hstore;
+
+DROP TABLE IF EXISTS flag CASCADE;
```

Comment:
You sure you want to drop the table if it exists? Maybe only create the table if it doesn't exist.

Reply:
I see this going two ways:
1) Create a new schema for each run. With this, the flags and features contained in their tables will all be associated with a single run
2) Add various runs to the same schema. In this case, you can differentiate each run by the `date_created` timestamp